dependencies.yaml

```
@@ -108,6 +108,20 @@ dependencies:
       match: REVISION:\ '\d+'
 
   # go-runner
+  - name: "registry.k8s.io/build-image/go-runner (go1.22-bullseye)"
```

Shouldn't this be `bookworm`?
```suggestion
  - name: "registry.k8s.io/build-image/go-runner (go1.22-bookworm)"
```